packages/liveblocks-react-ui/src/components/internal/AiComposer.tsx

```diff
@@ -168,12 +158,7 @@ export const AiComposer = forwardRef<HTMLFormElement, AiComposerProps>(
     const sendAiMessage = useSendAiMessage(chatId, {
       stream,
       copilotId,
-
-      // TODO: We shouldn't need to pass knowledge from AiChat to AiComposer
-      //       to useSendAiMessage, ideally it would be attached to a chat ID
-      //       behind the scenes inside AiChat.
-      knowledge: localKnowledge,
```

Comment:
I'm on board with this change in general, specifically this bit! Nice to have this removed 👍